src/analysis/normed_space/operator_norm.lean

```diff
@@ -1637,6 +1637,9 @@ have eq : _ := uniformly_extend_of_ind h_e h_dense f.uniform_continuous,
   end,
   cont := cont }
 
+lemma extend_eq (x : E) : extend f e h_dense h_e (e x) = f x :=
```

Comment:
Possibly a good `simp` lemma?

Reply:
The variant [dense_inducing.extend_eq](https://leanprover-community.github.io/mathlib_docs/topology/dense_embedding.html#dense_inducing.extend_eq) is also not `simp`. If we change it, then I think we should change both.

Reply:
That lemma would only be a conditional simp lemma because it needs `hf` and can't infer it from the LHS; so I don't think should be `simp`. Here it doesn't need that.

Reply:
good point